Lib/test/test_importlib/test_path.py

```diff
@@ -26,6 +27,17 @@ def test_reading(self):
 class PathDiskTests(PathTests, unittest.TestCase):
     data = data01
 
+    def test_package_spec_origin_is_None(self):
+        import pydoc_data
```

Comment:
I'm a little uneasy depending on the implementation details (presence, name, contents) of a built-in module. In the `util` module, there's a routine that constructs a package based on parameters. Perhaps something like that should be used to build a contrived package that matches the desired interface.